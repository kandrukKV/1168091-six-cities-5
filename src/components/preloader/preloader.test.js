import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import Preloader from "./preloader";

it(`Preloader is rendered correctly`, () => {
  const renderer = new ShallowRenderer();
  const tree = renderer.render(
      <Preloader/>
  );
  expect(tree).toMatchSnapshot();
});
