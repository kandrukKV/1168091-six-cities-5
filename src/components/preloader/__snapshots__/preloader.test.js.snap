// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader is rendered correctly 1`] = `
<h3
  className="near-places__title"
>
  Loading data...
</h3>
`;
